.env_template

```diff
@@ -0,0 +1,13 @@
+DATABASE_URL=postgres://db:db@db/db
+PG_FEATURESERV_SYSTEM_PORT=9000
+
+POSTGRES_DB=db
+POSTGRES_USER=db
+POSTGRES_PASSWORD=db
+DB_SYSTEM_PORT=5432
+
+PGHOST=db
+PGPORT=5432
+PGDATABASE=db
+PGUSER=db
+PGPASSWORD=db
```

Comment:
the env file has a lot of redundant values. 

I know the practice to define every value only once and the load them via the `environment` property. 

something like this would be easier IMHO (applies for the other serivces as well):

```yaml
  api:
    image: pramsey/pg_featureserv
    ports:
      - "0.0.0.0:${PG_FEATURESERV_SYSTEM_PORT}:9000"
   environment:
      - DATABASE_URL=postgres://${PGUSER}:${PGPASSWORD}@${PGHOST}/${PGDATABASE}
    depends_on:
      - db
```

docker compose automatically detects a `.env` file 

![image](https://github.com/JakobMiksch/osm_feature_info/assets/15704517/96ead0b6-3817-43d1-9b76-583911a97aa4)


see https://docs.docker.com/compose/environment-variables/set-environment-variables/

Reply:
yes this is better, updated